examples/search-index/searchIndexStaticFields.json

```diff
@@ -0,0 +1,65 @@
+{
+  "AWSTemplateFormatVersion":"2010-09-09",
+  "Description":"This template creates a Search Index with static mapping on the MongoDB Atlas API",
+  "Parameters":{
+    "ProjectId":{
+      "Type":"String",
+      "Description":"Project Id"
+    },
+    "ClusterName":{
+      "Type":"String",
+      "Description":"Cluster name"
+    },
+    "DatabaseName":{
+      "Type":"String",
+      "Description":"Database name"
+    },
+    "CollectionName":{
+      "Type":"String",
+      "Description":"Collection name"
+    },
+    "IndexName":{
+      "Type":"String",
+      "Description":"Index name"
+    },
+    "Profile":{
+      "Type":"String",
+      "Default":"default",
+      "Description":"Secret Manager Profile that contains the Atlas Programmatic keys"
+    }
+  },
+  "Resources":{
+    "MySearchIndex":{
+      "Type":"MongoDB::Atlas::SearchIndex",
+      "Properties":{
+        "Analyzer":"lucene.standard",
+        "Profile":{
+          "Ref":"Profile"
+        },
+        "ClusterName":{
+          "Ref":"ClusterName"
+        },
+        "CollectionName":{
+          "Ref":"CollectionName"
+        },
+        "Database":{
+          "Ref":"DatabaseName"
+        },
+        "ProjectId":{
+          "Ref":"ProjectId"
+        },
+        "Name":{
+          "Ref":"IndexName"
+        },
+        "Mappings":{
+          "Dynamic":false,
+          "Fields":[
+            "property_type:[{\"type\":\"autocomplete\"},{\"type\":\"string\"}]"
```

Comment:
apart from adjusting the example the documentation for the `Fields` property has not been updated.